app/routes/models/search/resultsModel.ts

```diff
@@ -170,6 +170,7 @@ export const resultsModel = async (query: any, error?: any) => {
   if(error) {
     return {
       ...defaultReturn,
+      relatedContentData: getRelatedContentLinks('results'),
```

Comment:
Currently, relatedContentData is only being passed if there is an error. It should go within the defaultReturn object so the links are available when there is an error as well as when there are results.

Reply:
Wow 🙈